assets/src/components/editing/elements/conjugation/ConjugationToolbar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { createButtonCommandDesc } from 'components/editing/elements/commands/commandFactories';
+import { CommandContext } from 'components/editing/elements/commands/interfaces';
+import { DescriptiveButton } from 'components/editing/toolbar/buttons/DescriptiveButton';
+import { Toolbar } from 'components/editing/toolbar/Toolbar';
+
+import * as ContentModel from 'data/content/model/elements/types';
+
+interface SettingsProps {
+  commandContext: CommandContext;
+  model: ContentModel.Conjugation;
+  toggleEdit: () => void;
+  editing: boolean;
+}
+
+export const ConjugationSettings = (props: SettingsProps) => {
+  return (
+    <Toolbar context={props.commandContext}>
```

Comment:
Should we add a delete option to all of these toolbars? I started adding one, then realized most (all?) don't so it wouldn't fit in and didn't add it, but it seems like it would be a good usability addition.